recipes/cornac/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cornac" %}
+{% set version = "0.1.0.post3" %}
```

Comment:
why post3? it seems post4 exists.

Reply:
@hmaarrfk thank you for pointing out.
Recipe updated to post4.